docs/enterprise-solutions/configuration/infrastructure-configuration/mcp/overview.mdx

```diff
@@ -0,0 +1,571 @@
+---
+title: "MCP Integration"
+sidebarTitle: "Overview"
+description: "Configure Model Context Protocol (MCP) servers and marketplace integrations for enterprise Cline deployments"
+---
+
+Model Context Protocol (MCP) provides standardized communication between AI models and external data sources, tools, and services. Enterprise MCP integration allows you to securely connect Cline to your organization's systems while maintaining governance and compliance.
+
+## Enterprise MCP Benefits
+
+<CardGroup cols={2}>
+  <Card title="Extensible Architecture" icon="puzzle-piece">
+    Connect to unlimited external tools, databases, APIs, and services through standardized MCP servers.
+  </Card>
+  
+  <Card title="Enterprise Security" icon="shield-alt">
+    Secure authentication, authorization, and audit trails for all MCP server communications.
+  </Card>
+  
+  <Card title="Centralized Management" icon="network-wired">
+    Manage and deploy MCP servers enterprise-wide with version control and configuration management.
+  </Card>
+  
+  <Card title="Compliance Ready" icon="clipboard-check">
+    Built-in logging, monitoring, and data governance for regulatory compliance requirements.
+  </Card>
+</CardGroup>
+
+## MCP Architecture Overview
+
+```mermaid
+graph TB
+    A[Cline Enterprise] --> B[MCP Hub]
+    B --> C[MCP Marketplace]
+    B --> D[Remote MCP Servers]
+    B --> E[Internal MCP Servers]
+    
+    C --> F[GitHub Integration]
+    C --> G[Slack Integration] 
+    C --> H[Jira Integration]
+    
+    D --> I[Custom APIs]
+    D --> J[Databases]
+    D --> K[Cloud Services]
+    
+    E --> L[Internal Tools]
+    E --> M[Legacy Systems]
+    E --> N[Security Systems]
+    
+    O[Enterprise Admin] --> B
+    P[Audit Logging] --> B
+    Q[Authentication] --> B
+```
+
```

Comment:
I like this mermaid graph a lot. Do you mind reviewing how you can present it currently on the documentation? It's barely visible and might add more confusion than clarification to the users visiting to this page.